data/story northern union.txt

```diff
@@ -139,9 +142,9 @@ mission "Meeting an Alpha 1"
 			apply
 				set "alpha lore 1: done"
 			`	The Alpha casually grabs Peg out of his seat, throws him across the cafe, and sits in the warlord's now vacant seat. "Oh, where do I begin? Let's start with our genesis."`
-			`	"I'm sure you know already that we were created as super soldiers to be used by various factions, but the reason for that is because those in charge were getting reports of normal soldiers refusing orders.  Things such as leveling towns full of innocent bystanders, shooting down unarmed passenger ships, contaminating spaceports with radiation or bioweapons, that sort of thing. The human soldiers were apparently brought up with the idea that 'life was sacred' or something." His tone turns to what you can only assume is mockery, "And that 'only bad people' should be killed, and when faced with the flimsy excuses for why they were slaughtering these innocents, they tended to refuse like the cowards they were."`
+			`	"I'm sure you know already that we were created as super soldiers to be used by various factions, but the reason for that is because those in charge were getting reports of normal soldiers refusing orders.  Things such as leveling towns full of innocent bystanders, shooting down unarmed passenger ships, contaminating spaceports with radiation or bioweapons, that sort of thing. The human soldiers were apparently brought up with the idea that 'life was sacred' or something." His tone turns to what you can only assume is mockery. "And that 'only bad people' should be killed. And when faced with the flimsy excuses for why they were slaughtering these innocents, they tended to refuse like the cowards they were."`
 			`	The being calls over a waiter, and whispers his order, before continuing. There is an interesting smell in the air, which both turns your stomach and compels you to be silent.`

```

Comment:
Remove the double space before "Things".

Reply:
Change:

> I'm sure you know already

To:

> I'm sure you already know

Reply:
Change:

> that sort of thing

To:

> That sort of things

Since before the "things" talked about are plural, and there's multiple things,

Reply:
Change:

> like the cowards they were.

To:

> like the cowards they are.

In this context, I'd imagine the one calling them cowards believes they still are cowards, so it should be present, not past.